src/heif.imageio/heifinput.cpp

```diff
@@ -76,7 +76,8 @@ heif_input_imageio_create()
     return new HeifInput;
 }
 
-OIIO_EXPORT const char* heif_input_extensions[] = { "heic", "heif", "heics",
+OIIO_EXPORT const char* heif_input_extensions[] = { "heic",  "heif", 
```

Comment:
There seems to be an extra space here that's tickling the clang-format failure:

```suggestion
OIIO_EXPORT const char* heif_input_extensions[] = { "heic",  "heif",
```